src/main/webapp/graphs_script.js

```diff
@@ -11,8 +11,9 @@ function drawGraph() {
     data.addColumn('string', 'Date');
     data.addColumn('number', 'Mood Rating');
     Object.keys(journalData).forEach((journal) => {
-        var date = new Date(journalData[journal].timestamp).toDateString();
-        data.addRow([date, journalData[journal].moodValue]);
+        var date = new Date(journalData[journal].timestamp);
+        var journalDate = (date.getMonth()+1) + "/" + date.getDate() + "/" + date.getFullYear() + "\n" + date.getHours() + ":" + date.getMinutes();
```

Comment:
If an entry is submitted between the first 9 minutes of an hour, the hour that is displayed looks like, for example, 1:5. It should be 1:05, so there should be some logic that verifies a single digit minute. Also, I think there should be one more description, such as AM or PM to add to the graph.